angr/analyses/reaching_definitions/atoms.py

```diff
@@ -21,22 +30,26 @@ class Tmp(Atom):
     """
     Represents a variable used by the IR to store intermediate values.
     """
-    __slots__ = ['tmp_idx']
+    __slots__ = ('tmp_idx', 'size', )
 
-    def __init__(self, tmp_idx):
+    def __init__(self, tmp_idx: int, size: int):
         super(Tmp, self).__init__()
         self.tmp_idx = tmp_idx
+        self._size = size
 
     def __repr__(self):
         return "<Tmp %d>" % self.tmp_idx
 
     def __eq__(self, other):
```

Comment:
Could annotate `-> bool`.

Reply:
I choose to not annotate default methods for fear of over-annotating. We can revisit this choice later on if it becomes a problem for mypy.